lib/config_windows.go

```diff
@@ -84,6 +84,15 @@ type Config struct {
 	// TODO: rename without cups_ prefix
 	NativePrinterPollInterval string `json:"cups_printer_poll_interval,omitempty"`
 
+	// Use the full username (joe@example.com) in job.
+	// TODO: rename without cups_ prefix
+	CUPSJobFullUsername *bool `json:"cups_job_full_username,omitempty"`
+
+	// When the job finishes spooling, mark it as done.
+	// This is for compatibility with other products that pause jobs to process them.
+	// TODO: this is only in the Windows build for now - decide whether useful in Unix
+	JobSpooledIsDone *bool `json:"job_spooled_is_done,omitempty"`
```

Comment:
Is there any reason why someone would want this to be false? It seems like you're fixing a bug here, not providing a new feature.

Unless you have reservations, let's remove this and just do it the right way always. That means you can revert a bunch of other stuff in this patch.